src/App.jsx

```diff
@@ -1,35 +1,46 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import * as React from "react";
 
-function App() {
-  const [count, setCount] = useState(0)
+const todolist = [
+  {
+    title:" Drink water",
+    objectID: 0,
+    id: "First:",
+  },
+  {
+    title: " Eat healthy food",
+    objectID: 1,
+    id:"Second:",
+  },
+  {
+  title: " Physical activity",
+  objectID: 2,
+  id:"Third:",
```

Comment:
You have an empty space as a first character in the three todo titles. Try to remove them. Could lead to bugs.